autogen.sh

```diff
@@ -7,10 +7,11 @@ export LC_ALL=C
 set -e
 srcdir="$(dirname $0)"
 cd "$srcdir"
+# shellcheck disable=SC2230
 if [ -z ${LIBTOOLIZE} ] && GLIBTOOLIZE="`which glibtoolize 2>/dev/null`"; then
```

Comment:
Does `command -v` work here?

Reply:
Yes, it does. Going to fix tonight.
Also in `command -v 2>/dev/null` the ` 2>/dev/null` is redundant, right?

Reply:
@dongcarl 
> Does `command -v` work here?

Fixed.